decentraland/unity-renderer
# feat: [BiW Editor] BiW should update Project metadata and thumbnail (Unity side)

## What this PR includes?

### Fixes #601 
BiW creates a Project in Builder, every time the Players edit a Scene that does not have a Project associated.

BiW must update the Project thumbnail of the project:
- [x] When the Project is created
- [x] When the player publishes the Scene
- [x] When the player exit the Scene without publishing changes

* Consider this last should only happen if there are unpublished changes
* Automatic screenshots will always be done using the reset camera view

## How to test it?

1. Go to: https://play.decentraland.zone/branch/feat/BiW-should-update-Project-metadata-and-thumbnail-Kernel-side/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/BiW-should-update-Project-metadata-and-thumbnail&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=80,154
2. Press 'K' to open the Builer In World editor and test the 3 use cases.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
